src/plugins/intel_gna/src/gna_plugin.cpp

```diff
@@ -665,145 +667,169 @@ void GNAPlugin::AddDebugProperties(const InferenceEngine::CNNLayerPtr layer,
 }
 #endif
 
+static void TransformModel(std::shared_ptr<ngraph::Function> model,
```

Comment:
This is static method which does not have any reference to plugin itself, maybe it would be good to move this to separate file to reduce size of gna_plugin.cpp

Reply:
Moved to separate file